automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/analysis/MethodConfigSpec.scala

```diff
@@ -12,10 +12,73 @@ import org.broadinstitute.dsde.workbench.service.test.WebBrowserSpec
 import org.scalatest.{FreeSpec, Matchers}
 
 import scala.collection.immutable.ListMap
+import scala.io.Source
 
-class MethodConfigSpec extends FreeSpec with Matchers with WebBrowserSpec with WorkspaceFixtures with UserFixtures
+class MethodConfigSpec extends FreeSpec with Matchers with WebBrowserSpec with WorkspaceFixtures with UserFixtures with DownloadFixtures
   with MethodFixtures with BillingFixtures with TestReporterFixture {
 
+  val wdl = """
+              |workflow w {
+              |  call t
+              |}
+              |
+              |task t {
+              |  String inString
+              |  String inWorkspaceRef
+              |  String inThisRef
+              |  Float inFloat
+              |  Int inInt
+              |  Boolean inBoolean
+              |  File inFile
+              |  Array[String] inStringArray
+              |  Array[String] inStringArray2
+              |  Map[String,String] inStringMap
+              |
+              |  command {
+              |    echo inString ${inString}
+              |    echo inWorkspaceRef ${inWorkspaceRef}
+              |    echo inThisRef ${inThisRef}
+              |    echo inFloat ${inFloat}
+              |    echo inInt ${inInt}
+              |    echo inBoolean ${inBoolean}
+              |    echo inFile ${inFile}
+              |    echo inStringArray ${sep=" " inStringArray}
+              |    echo inStringArray2 ${sep=" " inStringArray2}
+              |    echo inStringMap ${write_map(inStringMap)}
+              |  }
+              |  output {
+              |    String out = stdout()
+              |  }
+              |  runtime {
+              |    docker: "ubuntu"
+              |  }
+              |}
+              |""".stripMargin
+
+  val wdlInputsBase = ListMap(
+    "w.t.inString" -> "\"test\"",
+    "w.t.inFloat"-> "1.5",
+    "w.t.inInt" -> "2",
+    "w.t.inBoolean" -> "true",
+    "w.t.inFile" -> "\"gs://foo/bar\"",
+    "w.t.inStringArray" -> """["foo","bar"]""",
+    "w.t.inStringArray2" -> """["say \"hi\"!"]""",
+    "w.t.inStringMap" -> """{"foo":"bar"}"""
+  )
+
+  val refInputs = ListMap(
+    "w.t.inWorkspaceRef" -> """workspace.hello""",
```

Comment:
Don't need to use `"""` here. It's used above to avoid having to escape embedded `"` (or, in the case of `"""["say \"hi\"!"]"""`, to make it easier to see that `\"` represents the literal string `\"`, not an escaped `"`).